### PR TITLE
Add optional meta field to relationships (#99)

### PR DESCRIPTION
**Pull request by mperice**
Closes issue #99.

### CODE REVIEW

File: lib/fast_jsonapi/relationship.rb [hunk 1]
```diff
@@ -151,6 +155,15 @@ def add_links_hash(record, params, output_hash)
                                  end
     end
 
+    def add_meta_hash(record, params, output_hash)
+      output_hash[key][:meta] = case meta
+                                when Hash
+                                  meta
+                                when Proc
+                                  FastJsonapi.call_proc(meta, record, params)
+                                end
```
Comment (stas) [inline lib/fast_jsonapi/relationship.rb:164]: Minor suggestion, a simple `return ... if ...is_a?(Proc)` would have been enough though :see_no_evil:

File: README.md [hunk 2]
```diff
@@ -317,6 +317,23 @@ class MovieSerializer
...
+
+Meta can be defined either by passing a static hash or by using Proc to the `meta` key. In the latter case, the record and any params passed to the serializer are available inside the Proc as the first and second parameters, respectively.
+
+
+```ruby
+class MovieSerializer
+  include JSONAPI::Serializer
+
+  has_many :actors, meta: Proc.new do |movie_record, params|
+    {count: movie_record.actors.length }
```
Comment (ziggythehamster) [inline README.md:332]:
```suggestion
    { count: movie_record.actors.length }
```

**Comment by mperice** [comment]
Thanks @stas, @ziggythehamster for the suggestions ... have fixed and committed the changes.

**Comment by stas** [comment]
Merged! Thank you @mperice :bow:

**Pull request merged**